src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -639,7 +639,6 @@ protected virtual AccessibilityStates AccessibilityCalculateStates(ulong states)
             FlagSetter(ref accessibilityStates, AccessibilityStates.Highlightable, this.AccessibilityHighlightable);
             FlagSetter(ref accessibilityStates, AccessibilityStates.Focusable, this.Focusable);
             FlagSetter(ref accessibilityStates, AccessibilityStates.Focused, this.State == States.Focused);
-            FlagSetter(ref accessibilityStates, AccessibilityStates.Highlighted, this.IsHighlighted);
```

Comment:
Is "Highlighted" the only state which calculation is duplicated? AFAIK, we should override here only these states which value depends on NUI-specific properties.

Reply:
Found HIGHLIGHTABLE and FOCUSABLE are duplicated.

Reply:
HIGHLIGHTED is also duplicated, VISIBLE should also probably be removed from here, because DALi also calculates it (and does it better than always `true`),  DEFUNCT is also duplicated.

Reply:
Yes correct. I will updated. Thank you.